src/content/docs/licenses/license-information/general-usage-licenses/global-technical-support-offerings.mdx

```diff
@@ -632,7 +632,7 @@ If you need assistance with New Relic Products, you are in good hands with sever
     id="ticket"
     title="Support case"
   >
-    Support is now available in the New Relic platform! Just click on the question mark at the top right of your New Relic screen to surface contextual documentation and resources. Depending on the Service Level associated with your New Relic account, you may be eligible for ticketed support and can open a ticket without leaving the New Relic Platform. As an alternative, customers eligible for ticketed support may also open a support case from the [New Relic Support page](https://support.newrelic.com/)
+    Support is now available in the New Relic platform! Just click on the question mark at the top right of your New Relic screen to surface contextual documentation and resources. Depending on the Service Level associated with your New Relic account, you may be eligible for cas support and can open a case without leaving the New Relic Platform. As an alternative, customers eligible for case support may also open a support case from the [New Relic Support page](https://support.newrelic.com/)
```

Comment:
```suggestion
    Support is now available in the New Relic platform! Just click on the question mark at the top right of your New Relic screen to surface contextual documentation and resources. Depending on the Service Level associated with your New Relic account, you may be eligible for case support and can open a case without leaving the New Relic Platform. As an alternative, customers eligible for case support may also open a support case from the [New Relic Support page](https://support.newrelic.com/)
```

Reply:
I'll make a quick PR + publish this change myself

Reply:
This isn't actually on the docs site yet so I can't make a quick PR like I said! Will still need a fix